airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryReservedKeywords.java

```diff
@@ -0,0 +1,109 @@
+package io.airbyte.integrations.destination.bigquery;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+
+public class BigQueryReservedKeywords {
+
+  // https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#reserved_keywords
+  public static final List<String> RESERVED_KEYWORDS = ImmutableList.of(
```

Comment:
This class is not being used, but I created it for completeness.

Both this and the `SnowflakeReservedKeywords` file were copied from normalization: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/base-normalization/normalization/transform_catalog/reserved_keywords.py

Reply:
this is a useful comment to put in the code directly! Otherwise I can see a future developer deleting this class for being unused.

(you would probably want to link to the commit tree directly, since we're planning to nuke normalization from master. I.e. https://github.com/airbytehq/airbyte/blob/f226503bd1d4cd9c7412b04d47de584523988443/airbyte-integrations/bases/base-normalization/normalization/transform_catalog/reserved_keywords.py)